stable/bookstack/templates/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1beta2
```

Comment:
I think this can be apps/v1
https://github.com/helm/charts: "Supported Kubernetes Versions"
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deployment-v1-apps

Reply:
Started the base of this chart in June, must be a leftover! Fixed.